lib/t_web/controllers/mobile_auth_controller.ex

```diff
@@ -21,10 +21,13 @@ defmodule TWeb.MobileAuthController do
       token: Accounts.UserToken.encoded_token(token),
       user: render_user(user),
       # TODO proper screen_width
+      # TODO actually, should it be removed?
```

Comment:
@akhrail1996 I don't think profile from HTTP response to the auth request it's being used in any version of the app, right?

Reply:
@ruslandoga it's used for determining whether user is onboarded or not here: https://github.com/getsince/test5/blob/master/Since/Auth/AuthService.swift#L47

Reply:
after that - no, since Current.ownProfile is taken from profileChannel: 
https://github.com/getsince/test5/blob/master/Since/Auth/AuthedCoordinator.swift#L71
https://github.com/getsince/test5/blob/master/Since/App/AppSocket.swift#L105

Reply:
just in case, `user.onboarded_at` can be used to check if user has onboarded

Reply:
thanks, will explore that path